src/style.scss

```diff
@@ -60,10 +61,10 @@
 }
 
 .related-post .title {
-	margin: 0px;
+	margin: 0;
 	font-size: 1.5rem;
 }
 
 .related-post .excerpt {
-	margin: 0px;
+	margin: 0;
 }
```

Comment:
According to the design, the excerpt should be concatenated (with an ellipsis at the end of the line). This can be done with the following CSS:

```css
overflow: hidden;
text-overflow: ellipsis;
white-space: nowrap;
```